Las/Utils/JsonSerializer.cs

```diff
@@ -0,0 +1,91 @@
+using RestSharp;
+using Newtonsoft.Json;
+using RestSharp.Deserializers;
+using RestSharp.Serializers;
+
+namespace Lucidtech.Las.Utils
+{
+    /// <summary>
+    /// A Json publishes that allows the user to serialize and deserialize back and forth between serialized json objects
+    /// and deserialized general objects and specific Dictionaries.
+    /// </summary>
+    public class JsonSerialPublisher: ISerializer, IDeserializer
+    {
+        private Newtonsoft.Json.JsonSerializer _serializer;
+        
+        public string RootElement { get; set; }
+
```

Comment:
Is it conventional to have each member with newlines in between?
Should the public members be declared before private ones?

Reply:
No and Yes!